client/gui/CAnimation.h

```diff
@@ -122,7 +124,7 @@ class CAnimation
 
 	std::shared_ptr<IImage> getImage(size_t frame, size_t group=0, bool verbose=true) const;
 
-	void exportBitmaps(const boost::filesystem::path & path) const;
+	void exportBitmaps(const bfs::path & path, bool prependResourceName = false) const;

```

Comment:
I suggest do not make any changes in CAnimation.h and CAnimation.cpp as those functions are not used

Reply:
CAnimation.h and .cpp reverted